src/servers/CMakeLists.txt

```diff
@@ -229,6 +229,43 @@ install(
   RUNTIME DESTINATION bin
 )
 
+#
+# io_test_util (only build when TRTIS_ENABLE_GPU given the purpose)
```

Comment:
It is ok to only build when GPU enabled, but let's call it something non-test related since I think we can also use it as an example in the documentation at some point. Ideally make an examples/ sub-directory and put simple.cc in there and also this file and call it memory.cc or memory_alloc.cc.